uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -84,6 +85,8 @@ sealed class FeedTab(val name: String, val routePath: String) {
         object Podcast : FilteredFeed(FeedItem.Podcast::class, "Podcast", "podcast")
     }
 
+    val listState = LazyListState()
```

Comment:
If you do this, I think that it will have a static state, but is it difficult to keep it with remember etc.? 👀

Reply:
- It was difficult for me to keep using remember etc. in `FeedTab`.
- `remember { listState }` seems to work in `FeedList` Composable, but I'm not sure if it should be used here or what would improve it. 💦 
(`FeedList` の Composable では remember を使えますが、ここで使うべきなのか、使うと何がよくなるのか理解が浅いので判断できないでいます 💦 )

Reply:
Thank you. I don't know the best practice, but I can find a way to manage it using remember in Sample, but I haven't found any code to manage it elsewhere, so it's safer to manage it in Compose. I feel like it, but are you worried that it will be recomposed?
(ありがとうございます 🙏  自分もベストプラクティスはわからないですが、rememberを使って管理する方法はSampleで見つかるのですが、他の場所で管理させるコードはサンプルで見つかっていないので、どちらかというとComposeで管理させるほうが安全な気がしているのですが、recomposeされてしまうのではというのが懸念でしょうか？)
https://github.com/android/compose-samples/blob/26a103c414cc5447385756e04990af596e3fa91e/Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt#L100

Reply:
I see. 🤔 
There seems to be no problem in operation, so I using it with `remember {}`. https://github.com/DroidKaigi/conference-app-2021/pull/406/commits/a5e24bead4785dfe96addf2a21832b08fc4bfcf8

Reply:
How about managing it within Compose in this way? 👀 
https://github.com/DroidKaigi/conference-app-2021/commit/f03bd549cfc664b4b1dfd04bd210684804d6b2c5

Reply:
I see, there was a way to use Map!
Thanks! LGTM 🙆‍♂️